### DISCUSSION TITLE
How can I make modal tabs with Strapi?

### DISCUSSION THREAD

**Discussion by dobradovic** [strapi/strapi#8977] [Legacy Topics]
Hi, how can I make modal tabs with Strapi? I have already modal tabs(like bootstrap) in my Strapi page but cannot figure out how it is builded? Where I can find code for modal tabs that I have in my project? I have separated template with content for that modals but cant find where it is implemented? 

Thanks any advice would be helpful! :)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
